<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>203</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>VAIO Control Center</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="usesScrollButtons">
      <bool>true</bool>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab_battery">
      <attribute name="title">
       <string>Battery Charge</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="chk_battery_fast_charge">
         <property name="text">
          <string>Enable Battery Fast Charge</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frm_battery_protection">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QRadioButton" name="rad_battery_max_protection">
            <property name="text">
             <string>Maximum Battery Protection (50% charge)</string>
            </property>
            <attribute name="buttonGroup">
             <string>btngrp_battery_protection</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rad_battery_medium_protection">
            <property name="text">
             <string>Medium Battery Protection (80% charge)</string>
            </property>
            <attribute name="buttonGroup">
             <string>btngrp_battery_protection</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rad_battery_no_protection">
            <property name="text">
             <string>No Battery Protection (100% charge)</string>
            </property>
            <attribute name="buttonGroup">
             <string>btngrp_battery_protection</string>
            </attribute>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_kbd_bl">
      <attribute name="title">
       <string>Keyboard Backlight</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QFrame" name="frm_kbd_bl">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="verticalSpacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item row="3" column="0">
           <widget class="QLabel" name="lbl_kbd_timeout">
            <property name="text">
             <string>Timeout:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSlider" name="hslider_kbd_timeout">
            <property name="maximum">
             <number>3</number>
            </property>
            <property name="tracking">
             <bool>false</bool>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="invertedControls">
             <bool>false</bool>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBelow</enum>
            </property>
            <property name="tickInterval">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="lbl_kbd_timeout_val">
            <property name="text">
             <string>10 seconds</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QCheckBox" name="chk_enable_kbd_bl">
            <property name="text">
             <string>Enable Keyboard Backlight</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_touchpad">
      <attribute name="title">
       <string>Touchpad</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QCheckBox" name="chk_enable_touchpad">
         <property name="text">
          <string>Enable Touchpad</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_als">
      <attribute name="title">
       <string>ALS</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QFrame" name="frm_als">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="chk_enable_als_power">
            <property name="text">
             <string>Enable Ambient Light Sensor</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="lbl_als_lux">
              <property name="text">
               <string>Luminance:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="lbl_als_lux_val">
              <property name="text">
               <string>ALS Luminance</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="lbl_als_kelvin">
              <property name="text">
               <string>Temperature:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="lbl_als_kelvin_val">
              <property name="text">
               <string>ALS Temperature</string>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_lid">
      <attribute name="title">
       <string>Lid Resume Control</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QFrame" name="frm_lid">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="chk_lid_s3">
            <property name="text">
             <string>Resume from suspend on lid open</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chk_lid_s4">
            <property name="text">
             <string>Resume from hibernate on lid open</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_optdev">
      <attribute name="title">
       <string>Optical Device</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_5">
       <item row="0" column="0">
        <widget class="QCheckBox" name="chk_enable_optdev">
         <property name="text">
          <string>Enable Optical Device</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_thermal">
      <attribute name="title">
       <string>Thermal Profile</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QFrame" name="frm_thermal">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QRadioButton" name="rad_thermal_balanced">
            <property name="text">
             <string>Balanced Profile</string>
            </property>
            <attribute name="buttonGroup">
             <string>btngrp_thermal</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rad_thermal_silent">
            <property name="text">
             <string>Silent Profile</string>
            </property>
            <attribute name="buttonGroup">
             <string>btngrp_thermal</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rad_thermal_performance">
            <property name="text">
             <string>Performance Profile</string>
            </property>
            <attribute name="buttonGroup">
             <string>btngrp_thermal</string>
            </attribute>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="btngrp_thermal"/>
  <buttongroup name="btngrp_battery_protection"/>
 </buttongroups>
</ui>
